<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>812</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Quake II PAK File Extractor</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>571</width>
     <height>78</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btn_select_output_folder">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select Folder for Ouptut of Extracted Files</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_output_folder">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>200</y>
     <width>571</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="btn_select_pak_file">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select PAK File</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_pak_file">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_extract_all">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>720</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Extract ALL Files</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lst_files">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>310</y>
     <width>571</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_extract_selected">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>650</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Extract Selected File</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>211</width>
     <height>381</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>quake2_box.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
